src/rest/QueryResult.ts

```diff
@@ -105,6 +105,12 @@ export interface IQueryResult {
    * See also [Coveo Machine Learning](http://www.coveo.com/go?dest=cloudhelp&lcid=9&context=177).
    */
   isRecommendation: boolean;
+
+  /**
+   * Contains a value that specifies whether the result was seen by the user targeted in the query.
+   */
+  isUserActionView?: boolean;
```

Comment:
What do you think of `viewedByUser` or `seenByUser` as the name of this property?

Reply:
Same, come from the searchApi, so I don't think it'd be really useful or nice